### ISSUE TITLE
Site language selector shows even when empty/not set

### ISSUE THREAD

**Issue by Vasperous** [squidfunk/mkdocs-material#2095]
- [x] I've read the [contribution guidelines][1] and agree with them

__I've found a bug and checked that ...__

- [x] ... the problem doesn't occur with the default MkDocs template
- [x] ... the problem is not in any of my customizations (CSS, JS, template)
- [x] ... the documentation does not mention anything about my problem
- [x] ... there are no open or closed issues that are related to my problem

## Description

Using Mkdocs v1.1.2 and Mkdocs Material v6.1.7 (with Insiders v1.11.1, `6.1.7+insiders.1.11.1`) the header is showing the language selection button even when the selection is empty.  When the user hovers over the language icon it shows a small empty list.

![image](https://user-images.githubusercontent.com/8703632/101523682-e95ba480-3956-11eb-8b32-160814155a82.png)

### Expected behavior

If the language site selector is not set in the `mkdocs.yml` then the language selection icon next to the search bar should not appear.

### Actual behavior

Language site selector appears regardless of the configuration in the `mkdocs.yml`.

### Steps to reproduce the bug

1. Create a new Mkdocs project
2. Apply default Material Mkdocs configuration
3. Serve using `mkdocs serve`

### Package versions

* Python: `python --version`
  * Python 3.8.5
* MkDocs: `mkdocs --version`
  * mkdocs, version 1.1.2
* Material: `pip show mkdocs-material | grep -E ^Version`
  * Version: 6.1.7+insiders.1.11.1

### Project configuration

``` yaml
site_name: My Docs

# If serving from a website you would want 'true', for offline you want false
use_directory_urls: false

theme:
  name: 'material'
  custom_dir: theme
  font:
    text: 'Roboto'
    code: 'Roboto Mono'
  language: 'en'
  features:
    - navigation.tabs
    - navigation.instant
    - search.highlight
    - search.suggest
    - header.hide
    - toc.autohide

extra:
  generator: false

markdown_extensions:
  - admonition
  - attr_list
  - def_list
  - footnotes
  - pymdownx.details
  - pymdownx.emoji:
      emoji_index: !!python/name:materialx.emoji.twemoji
      emoji_generator: !!python/name:materialx.emoji.to_svg
  - pymdownx.highlight:
      use_pygments: true
  - pymdownx.inlinehilite
  - pymdownx.keys
  - pymdownx.superfences
  - pymdownx.tabbed
  - toc:
      permalink: '#'
      toc_depth: 5

plugins:
  - search:
      prebuild_index: true
```

### System information

* OS:
  * Mac OS Version 11.0.1
* Browser:
  * Brave v1.17.75

  [1]: https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md

**Comment by squidfunk** [comment]
Confirmed, thanks for reporting!

**Comment by squidfunk** [comment]
Fixed in Insiders 1.12.1 😊

**Comment by Vasperous** [comment]
@squidfunk  thank you!!!
